utils/copy-db-date.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var db = require('../db');
+
+module.exports = function (date) {
+	return new db.Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate());
```

Comment:
To avoid erroneous usage it might be good to require here instances of `db.Date` (best via `instanceof`)

It will also help to write accurate tests ;-)